### PR TITLE
chore(readme): update to beta

### PR DESCRIPTION
**Pull request by nothingismagick**
**What kind of change does this PR introduce?** (check at least one)
- [x] Other, please describe:

README update for new features, libs and structure.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,8 @@ Tauri Apps
...
 | [**cli.rs**](https://github.com/tauri-apps/tauri/tree/dev/tooling/cli.rs)   | create, develop and build apps | [![](https://img.shields.io/crates/v/tauri-cli.svg)](https://crates.io/crates/tauri-cli)         | ✅   | ✅   | ✅   |
-| [**cli.js**](https://github.com/tauri-apps/tauri/tree/dev/tooling/cli.js)   | Node.js CLI wrapper for cli.rs | [![](https://img.shields.io/npm/v/tauri.svg)](https://www.npmjs.com/package/tauri)               | ✅   | ✅   | ✅   |
+| [**cli.js**](https://github.com/tauri-apps/tauri/tree/dev/tooling/cli.js)   | Node.js CLI wrapper for cli.rs | [![](https://img.shields.io/npm/v/@tauri-apps/cli.svg)](https://www.npmjs.com/package/@tauri-apps/cli)               | ✅   | ✅   | ✅   |
+| [**api.js**](https://github.com/tauri-apps/tauri/tree/dev/tooling/cli.js)   | JS API for interaction with Rust backend | [![](https://img.shields.io/npm/v/@tauri-apps/api.svg)](https://www.npmjs.com/package/@tauri-apps/api)               | ✅   | ✅   | ✅   |
```
Comment (lucasfernog) [inline README.md]: wrong github url here, should be just /api

File: README.md [hunk 2]
```diff
@@ -33,7 +34,7 @@ Tauri Apps
 ## Introduction
 Tauri is a framework for building tiny, blazing fast binaries for all major desktop platforms. Developers can integrate any front-end framework that compiles to HTML, JS and CSS for building their user interface. The backend of the application is a rust-sourced binary with an API that the front-end can interact with.
 
-The user interface in Tauri apps currently leverages Cocoa/WebKit on macOS, gtk-webkit2 on Linux and MSHTML (IE10/11) or Webkit via Edge on Windows. **Tauri** uses (and contributes to) the MIT licensed project that you can find at [webview](https://github.com/webview/webview) and the [related webview organization](https://github.com/webview).
+The user interface in Tauri apps currently leverages [`winit`](https://docs.rs/winit) as a WebView abstraction for all platforms via the **Tauri-team** incubated and maintained [wry](https://github.com/tauri-apps/wry), which creates a unified interface to these system webviews.
```
Comment (lucasfernog) [inline README.md]: winit is a window lib, maybe we should note that we use gtk on linux and still mention wkwebview/webview2/webkitgtk

File: README.md [hunk 3]
```diff
@@ -46,13 +47,13 @@ If you are interested in making a tauri-app, please visit the [documentation web
 - [x] Build on Windows (.exe, .msi)
 - [x] Copy Buffer
 - [x] Device Notifications (toast)
-- [ ] Self Updater (in progress)
+- [x] Self Updater
+- [x] App Signing (coming soon)
```
Comment (lucasfernog) [inline README.md]: why coming soon? they're all published

File: README.md [hunk 4]
```diff
@@ -129,16 +130,17 @@ Tauri is a system composed of a number of moving pieces:
...
 
 ### Major Runtimes
 - node.js for running the CLI (deno and pure rust are on the roadmap)
 - cargo for testing, running the dev service, building binaries and as the runtime harness for the webview
 
 ### Major Languages
-- typescript for the CLI
-- javascript for the JS-side of the API
+- rust for the CLI
+- typescript bindings to the RUST Cli
```
Comment (lucasfernog) [inline README.md]: CLI or Cli?

File: README.md [hunk 5]
```diff
@@ -129,16 +130,17 @@ Tauri is a system composed of a number of moving pieces:
...
+- typescript the JS-side of the API
```
Comment (lucasfernog) [inline README.md]: "typescript for the JS-side"?

File: README.md [hunk 6]
```diff
@@ -34,7 +34,7 @@ Tauri Apps
 ## Introduction
 Tauri is a framework for building tiny, blazing fast binaries for all major desktop platforms. Developers can integrate any front-end framework that compiles to HTML, JS and CSS for building their user interface. The backend of the application is a rust-sourced binary with an API that the front-end can interact with.
 
-The user interface in Tauri apps currently leverages [`winit`](https://docs.rs/winit) as a WebView abstraction for all platforms via the **Tauri-team** incubated and maintained [wry](https://github.com/tauri-apps/wry), which creates a unified interface to these system webviews.
+The user interface in Tauri apps currently leverages [`winit`](https://docs.rs/winit) as a WebView abstraction for all platforms via the **Tauri-team** incubated and maintained [wry](https://github.com/tauri-apps/wry), which creates a unified interface to these system webviews using gtk on linux and leveraging wkwebkit (macOS), webview2 (windows) webkitgtk (linux).
```
Comment (lucasfernog) [inline README.md]:
```suggestion
The user interface in Tauri apps currently leverages [`winit`](https://docs.rs/winit) as a window handling library on macOS and Windows, and [`gtk`](https://gtk-rs.org/docs/gtk/) on Linux via the **Tauri-team** incubated and maintained [WRY](https://github.com/tauri-apps/wry), which creates a unified interface to the system webview, leveraging WebKit on macOS, WebView2 on Windows and WebKitGTK on Linux.
```

**Comment by nothingismagick** [comment]
> what's the ]

dunno what you mean @lucasfernog

File: README.md [hunk 7]
```diff
@@ -33,26 +34,33 @@ Tauri Apps
...
 - [x] Build on MacOS (.app, .dmg)
 - [x] Build on Linux (.deb, AppImage)
 - [x] Build on Windows (.exe, .msi)
 - [x] Copy Buffer
 - [x] Device Notifications (toast)
-- [ ] Self Updater (in progress)
+- [x] Self Updater
+- [x] App Signing
+- [x] Frameless Mode]
+- [x] Transparent Mode]
```
Comment (jmannanc) [inline README.md]: Extra `]` at the end of these 2 lines

File: README.md [hunk 8]
```diff
@@ -33,26 +34,33 @@ Tauri Apps
...
 - [x] App Icons
 - [x] Build on MacOS (.app, .dmg)
 - [x] Build on Linux (.deb, AppImage)
 - [x] Build on Windows (.exe, .msi)
 - [x] Copy Buffer
 - [x] Device Notifications (toast)
-- [ ] Self Updater (in progress)
+- [x] Self Updater
+- [x] App Signing
+- [x] Frameless Mode]
```
Comment (lucasfernog) [inline README.md]:
```suggestion
- [x] Frameless Mode
```

File: README.md (hunk 7 above)
Comment (lucasfernog) [inline README.md]:
```suggestion
- [x] Transparent Mode
```

**Review by lucasfernog — APPROVED** [review]

File: README.md [hunk 9]
```diff
@@ -129,16 +137,17 @@ Tauri is a system composed of a number of moving pieces:
...
 - node.js for running the CLI (deno and pure rust are on the roadmap)
 - cargo for testing, running the dev service, building binaries and as the runtime harness for the webview
 
 ### Major Languages
-- typescript for the CLI
-- javascript for the JS-side of the API
-- rust for bindings, rust side of the API, harnesses
-- C / obj-c for interfacing with the webviews
+- Rust for the CLI
+- EcmaScript bindings to the Rust API, written in typescript
```
Comment (nklayman) [inline README.md:148]: Should we say JavaScript instead of EcmaScript here to make it more clear for beginners?

File: README.md [hunk 10]
```diff
@@ -160,35 +169,24 @@ Test all the things! We have a number of test suites, but are always looking to
...
+Tauri aims to be a sustainable collective based on principles that guide [sustainable free and open software communities](https://sfosc.org). To this end it has become a Programme within the [Commons Conservancy](https://commonsconservancy.org/), and you can contribute financially via [Open Collective](https://opencollective.com/tauri).
 
 ## Contributing
 Please make sure to read the [Contributing Guide](./.github/CONTRIBUTING.md)
 before making a pull request.
 
 Thank you to all the people who already contributed to Tauri!
 
-Special thanks to the development team at Volentix Labs for the encouragement and support in the early phases of Tauri, notably Rhys Parry and Gregory Luneau.
-
```
Comment (Beanow) [inline README.md:181]: Backtracking a bit, but two points:
- Have the removed special thanks become obsolete?
- Technically I think we're in the "setup phase" https://dracc.commonsconservancy.org/0020/ before formally going into effect as a programme. May be running a bit ahead of the actual status there.

**Pull request merged**